lib/watch-files.js

```diff
@@ -1,19 +1,39 @@
 const chokidar = require('chokidar');
-const { isEmpty } = require('omnibelt');
+const { isEmpty, forEachSerialP, includes  } = require('omnibelt');
 const { logError, logResult, loadConfig, unlockConfig } = require('./utils');
 const paramsByCommand = require('./get-upload-params');
 const getUploader = require('./get-uploader');
 const path = require('path');
 
 module.exports = (nameOfDirectory) => {
   const uploader = getUploader(paramsByCommand[nameOfDirectory], { usePolling: process.platform === 'win32' });
+  const queue = [];
+  const queueData = new Map();
+  let isRunning = false;
+
   return async (command) => {
     const config = await loadConfig();
     if (isEmpty(config)) {
       return;
     }
+    const timerId = setInterval(async () => {
+      if (queue.length && !isRunning) {
+        logResult('upload', 'processing queue', 'magentaBright');
+        isRunning = true;
+        await forEachSerialP(async () => {
+          const file = queue.pop();
+          const { qPattern, qCommand, qConfig } = queueData.get(file);
```

Comment:
This does not match what you set on the map below line 61. I think you only need pattern, command and config will always be the same.

Reply:
Thank you that was staring me in the face and I couldn't find what was wrong 🙏 👏